subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -598,12 +598,29 @@ config BT_CTLR_DYNAMIC_INTERRUPTS
 	  permit use of SoC's peripheral for custom use when Bluetooth is not
 	  enabled.
 
+choice
+	prompt "Optimazation options"
+	default BT_CTLR_OPTIMIZE_FOR_SPEED
+
+config BT_CTLR_LTO
+	bool "Link Time Optimization"
+	depends on ARM
+	select SHARED_INTERRUPTS
+	select ISR_TABLES_LOCAL_DECLARATION
+	select LTO
+	help
+	  Enable use of Link Time Optimizations, to reduce code space usage.
+	  This disables compiler optimization for execution speed.
+
 config BT_CTLR_OPTIMIZE_FOR_SPEED
-	prompt "Optimize for Speed" if !(SOC_SERIES_NRF51X && BT_CTLR_LE_ENC)
-	bool
-	default y if BT_CTLR_LE_ENC
+	bool "Optimize for Speed"
```

Comment:
Should this depend on `!LTO`? 

Otherwise it is possible to set `CONFIG_BT_CTLR_LTO=n` but `CONFIG_LTO=y`

Similarly, should `BT_CTLR_LTO` depend on `LTO` intead of `select`ing it?